### PR TITLE
Add Pony layer

### PR DESCRIPTION
**Pull request by 0x1997**
Add a layer for the [Pony](http://www.ponylang.org/) programming language.

### CODE REVIEW

File: layers/+lang/pony/packages.el [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    flycheck
+    (flycheck-pony :location
+                   (recipe
+                    :fetcher github
+                    :repo "rmloveland/flycheck-pony"))
+    ponylang-mode
+    (pony-snippets :location
+                   (recipe
+                    :fetcher github
+                    :repo "SeanTAllen/pony-snippets"))
```
Comment (jmiven) [inline layers/+lang/pony/packages.el]: =flycheck-pony= and =pony-snippets= are now in MELPA.

Comment (0x1997) [inline layers/+lang/pony/packages.el]: Thanks. Updated.

**Comment by 0x1997** [comment]
@d12frosted Updated.

**Comment by tchajed** [comment]
Ping. @0x1997 could you rebase on top of the latest develop and resolve the conflicts in `LAYERS.org`? It would be great to see this merged!

**Comment by 0x1997** [comment]
@tchajed I've updated the pony-layer branch. Should I reopen the PR?

**Comment by tchajed** [comment]
The PR is still open, you don't have to do anything on GitHub.

Have you re-tested this against the latest version of Spacemacs? I started working on it and it looks like some of the interfaces have changed (for example, I couldn't get flycheck to work with just the layer installed).

**Comment by robbyoconnor** [comment]
You should resolve the merge conflicts.

**Pull request closed without merging**